code/obj/item/mops_cleaners.dm

```diff
@@ -1346,6 +1369,23 @@ TYPEINFO(/obj/item/handheld_vacuum/overcharged)
 			return
 		. = ..()
 
+	syndicate
+		name = "\improper GR1-Mk3 back tank"
+		desc = "A little label on the side reads \"Now with corrosive substances support!\"."
+		icon_state = "janitor_tank_s"
+		item_state = "janitor_tank_s"
+		initial_volume = 500
+		initial_reagents = list("sewage" = 500)
```

Comment:
Might be fun if it was full of a random mixture of industrial waste style chemicals: sewage, "yuck", carbon, grease, oil, toxic slurry etc.